tests/lit-tests/loop_nounroll.ispc

```diff
@@ -0,0 +1,99 @@
+// Test to check '#pragma nounroll' functionality for different loop statemnts.
+
+// RUN: %{ispc} %s --target=avx2-i32x8 --nowrap -O0 --emit-llvm-text -o - | FileCheck %s -check-prefix=CHECKO0
```

Comment:
If no stdlib is used in the test, it's better to have ``--nostdlib`` in the arguments - it speeds up the compilation.

Reply:
Done